pkg/configuration/configuration.go

```diff
@@ -96,6 +112,10 @@ type TestConfiguration struct {
 	// AcceptedKernelTaints
 	AcceptedKernelTaints []AcceptedKernelTaintsInfo `yaml:"acceptedKernelTaints,omitempty" json:"acceptedKernelTaints,omitempty"`
 	SkipHelmChartList    []SkipHelmChartList        `yaml:"skipHelmChartList" json:"skipHelmChartList"`
+	// SkipScalingTestDeploymentNames
+	SkipScalingTestDeploymentNames []SkipScalingTestDeploymentNamesInfo `yaml:"skipScalingTestDeploymentNames,omitempty" json:"skipScalingTestDeploymentNames,omitempty"`
```

Comment:
Same suggestion for the actual yaml fields: I think the "Names" part could be omitted: what about `json:skipScalingTestDeployments,omitempty` and `json:skipScalingTestStatefulSets,omitempty`?